services/parent/pom.xml

```diff
@@ -95,6 +95,12 @@
       <plugin>
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-maven-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <systemProperties>
+            <vertx.json.base64>legacy</vertx.json.base64>
+          </systemProperties>
```

Comment:
maybe we should also set the logging manager here?
```xml
<java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
```

Reply:
wasn't needed for local services to run (base64 mode is mandatory) 
but, ok, I'll add it in sake of consistensy